### DISCUSSION TITLE
Has anyone had luck forcing FPS above 20? How do you do it on HA?

### DISCUSSION THREAD

**Discussion by thatdaveguy1** [mrlt8/docker-wyze-bridge#1073] [Q&A] [answered]
I used to use Docker and I could force the FPS up to 25 which had some success. 

But I’ve not had any luck figuring out the command on Home Assistant?

**Comment by mrlt8** [comment]
FPS adjustments seem to mostly effect the shutter speed of the camera, but you should be able to get and set it using the REST API or MQTT:

https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Commands

**Comment by thatdaveguy1** [comment]
> FPS adjustments seem to mostly effect the shutter speed of the camera, but you should be able to get and set it using the REST API or MQTT:
> 
> https://github.com/mrlt8/docker-wyze-bridge/wiki/Camera-Commands

Ah thanks, the MQTT stuff has proven to be too difficult for me to figure out so far

**Reply by mrlt8** [reply, marked as answer]
You should be able to use the http api with the fps endpoint:
get value:
```
http://homeassistant.local:5000/api/<cam-name>/fps
```
set value:
```
http://homeassistant.local:5000/api/<cam-name>/fps/25
```

or if you enable MQTT in home assistant, then you can adjust it in the diagnostic menu:
<img width="331" alt="has" src="https://github.com/mrlt8/docker-wyze-bridge/assets/67088095/e440edf7-b9b5-453a-a9f4-c411e6049dcc">

**Reply by thatdaveguy1** [reply]
Thank you! The diagnostic menu worked great
